### DISCUSSION TITLE
Amazon Account Blocked!

### DISCUSSION THREAD

**Discussion by harrydaws** [alandtse/alexa_media_player#1587] [Q&A] [answered]
Hello, I had set up this integration and was all working fine but now I have lost access to my amazon account becasue I have no way of receiving a verification code. Is there any way to retrieve the 2fa secret from home assistant so I can paste it into an authenticator app.

Thanks,
Harry

**Comment by alandtse** [comment, marked as answer]
It's be in your /config/.storage/core.config_entries files under the alexa_media config entry if you had saved it. Otherwise no.
```json
"data": {
                    "otp_secret":"",
}
